API.Services/Services.Persons/PersonService.cs

```diff
@@ -0,0 +1,98 @@
+﻿using API.Data;
+using API.Models;
+using API.Models.Models.Persons;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace API.Services.Services.Persons
+{
+    public class  PersonService
+    {
+        private readonly Guid _userID;
```

Comment:
It might be easier to put in ctx = new ApplicationDbContext(); here, since you use it in all of your methods. just a thought ;)

Reply:
We might be able to do this @DocMTyler . When we test we should definitely check